index.html

```diff
@@ -159,22 +152,22 @@
 			<!-- global class of "wrapper" is applied -->
 			<!-- "align-text" is a home-page class to center the text -->
 			<div class="hero-home wrapper align-text">
-				<h3>Preserving Tomorrow by Building Today</h3>
+				<h1 class="h3-styles">Preserving Tomorrow by Building Today</h1>
 				<p>
 					If These Lands Could Talk (ITLCT) is global network of change-makers,
-					supporters, and allies who support native and indigenous communities
+					supporters, and allies who support Native and Indigenous communities
 					to thrive authentically and on their own terms with digital technology
 					both now and in the future.
 				</p>
 				<!-- about us "button" visible on mobile style only -->
 				<div>
-					<button class="button3">About US</button>
+					<a href="#"><button class="button3">About US</button></a>
 				</div>
 				<!-- "btn-container" holds the other two buttons for the hero section -->
 				<!-- these buttons are "hidden" for mobile and only visible for tablet and larger -->
 				<div class="hidden btn-container">
-					<button class="button1">Our Impact</button>
-					<button class="button1">Our Values</button>
+					<a href="#"><button class="button1">Our Impact</button></a>
```

Comment:
When you have a moment, can you please update this to the impact and values html files?

Reply:
resolved this issue